src/Config/ConfigInterface.php

```diff
@@ -52,6 +52,13 @@ public function getEntryPoints(): array;
      */
     public function getAssetFiles(): array;
 
+    /**
+     * Return a list of files to exclude when bundling. These are the files defined under 'excludes'.
+     *
+     * @return string[]
+     */
+    public function getExcludedImports(): array;
```

Comment:
TL;DR; rename `getExcludedImports` to `getExcludedFiles`

Ideally we'd specify imports in the config, i.e. `jquery`, or `@angular/core`. But that is hard to implement as of now, since
- the file resolvers are created by the plugins, thus not accessible for the `PipelineBundler`
- this functionality is not only for js, but also for css

Hence specifying files relative to the project root is the best we can do.